E2-Breakdown-Money/src/main/java/com/redelles/e2/BreakDownMoneyMainProgramD.java

```diff
@@ -14,5 +14,12 @@ public static void main(String[] args) throws IOException {
          * x.add()
          * x.get(position)
          */
+
+        Integer amount = Numbers.getAmount();
+        Integer[] quantities = new Integer[3];
```

Comment:
quantities = new Integer[3] --> creates a new instance
quantities = Utils.getArrayOfThree() -- returns another instance and is set to the variable

The first sentence has not sense because you are overwriting his value in the secon